cmds/exp/nvme_unlock/nvme_unlock.go

```diff
@@ -69,6 +70,20 @@ func getSysfsInfo(index string, field string) (string, error) {
 }
 
 func run(disk string, verbose bool, verboseNoSanitize bool, noRereadPartitions bool, lock bool) error {
+	if disk == "" {
+		disks, err := finddrive.FindSlotType(finddrive.M2MKeySlotType)
+		if err != nil {
+			return fmt.Errorf("error finding boot disk: %v", err)
+		}
+		if len(disks) == 0 {
+			return fmt.Errorf("no boot disk found")
+		}
+		disk = disks[0]
+		if len(disks) > 1 {
+			log.Printf("Multiple boot disk candidates found, using the first from the list: %v", disks)
```

Comment:
Consider printing all disks found, which can be helpful in such rare cases

Reply:
It does print all disks found.  I rephrased the message to make this more clear.